compiler/ballerina-lang/src/main/java/io/ballerina/projects/ModuleContext.java

```diff
@@ -42,6 +42,7 @@
 import org.wso2.ballerinalang.compiler.util.CompilerOptions;
 import org.wso2.ballerinalang.programfile.CompiledBinaryFile;
 import org.wso2.ballerinalang.programfile.PackageFileWriter;
+import org.testing.Assert;
```

Comment:
@someshfengde This not the correct java package. Please run ./gradlew build in the project root folder to validate whether the build is successfully passing locally

Reply:
yes sure